tests/cache/tests.py

```diff
@@ -1473,6 +1474,14 @@ def test_empty_cache_file_considered_expired(self):
             self.assertIs(cache._is_expired(fh), True)
 
 
+class FileBasedCachePathLibTests(FileBasedCacheTests):
+    def setUp(self):
+        super().setUp()
+        for cache_params in settings.CACHES.values():
+            cache_params.update({'LOCATION': Path(self.dirname)})
```

Comment:
```python
cache_params['LOCATION'] = Path(self.dirname)
```

Reply:
Good call. Fixed in the latest revision.